src/transformers/models/bert/modeling_bert.py

```diff
@@ -451,7 +452,11 @@ def __init__(self, config):
         if self.add_cross_attention:
             if not self.is_decoder:
                 raise ValueError(f"{self} should be used as a decoder model if cross attention is added")
-            self.crossattention = BertAttention(config)
+            # cross attention cannot have relative position embeddings
+            cross_attention_config = copy.deepcopy(config)
+            cross_attention_config.position_embedding_type = "absolute"
+
+            self.crossattention = BertAttention(cross_attention_config)
```

Comment:
This is a bit hackish. If we start to turn this argument on and off, maybe it should be an init argument? It can default to `None` in which case we take the config value (so this is 100% backward compatible).

Reply:
yeah agree